cmd/root.go

```diff
@@ -59,6 +59,7 @@ var prometheusMetricsOptions = sreProvider.PrometheusOptions{
 var discoveryPrometheusOptions = common.PrometheusOptions{
 	Names:    envStringExpand("PROMETHEUS_NAMES", ""),
 	URL:      envStringExpand("PROMETHEUS_URL", ""),
+	Auth:     envStringExpand("PROMETHEUS_HTTP_AUTH", ""),
```

Comment:
Bad idea IMHO use two separate lists for prometheuses and auth.
Better use one list in format "user:pass@url" separated by commas where "user:pass@" part is optional